dlt/pipeline/pipeline.py

```diff
@@ -617,6 +617,36 @@ def last_trace(self) -> PipelineTrace:
             return self._last_trace
         return load_trace(self.working_dir)
 
+    def last_pipeline_step_trace(self, step_name: TPipelineStep) -> PipelineStepTrace:
```

Comment:
this is good. what I do not like it is that it should be a part of `PipelineTrace` class. we have plenty of methods and code in pipeline already and props below have nothing to do with pipeline itself...